src/imperative/imperative.cc

```diff
@@ -501,6 +501,10 @@ std::vector<NDArray*> Imperative::Backward(
     }
   }
 
+  if (dmlc::GetEnv("MXNET_MEM_PLAN_VERBOSE_LOGGING", false)) {
```

Comment:
nice hack :)

Reply:
Can you explain how logging of static memory helps here ?

Reply:
He wants to dump the graph. maybe should be separate PR?

Reply:
It's actually dumping out the computation graph.

Reply:
@apeforest isn't that what I wrote?

Reply:
@larroy I did not see your comment before I refresh the page.  Our minds think alike :)
Sure, I can separate this into a different PR. I added here only to help @kshitij12345 dump out the graph and understand the backward pass better.

Reply:
The PR is great, up to you to separate not a big deal. I'm not a radical on my reviews.

Reply:
@apeforest Really like the dump, its great help to actually see the graph. Thank You.